src/settings/descriptors/kpi/kpiOnHoverDescriptor.ts

```diff
@@ -89,6 +90,14 @@ export class KpiOnHoverDescriptor extends KpiBaseDescriptor {
         this.currentValueFontSize.value = this.getValidFontSize(this.currentValueFontSize.value);
     }
 
+    public processHighContrastMode(colorPalette: ISandboxExtendedColorPalette): void {
+        super.processHighContrastMode(colorPalette);
+
+        const isHighContrast: boolean = colorPalette.isHighContrast;
+        this.currentValueColor.visible = isHighContrast ? false : this.currentValueColor.visible;
```

Comment:
I don't think we need this logic, since you add currentValueColor to slices and do all of this in the Base method. Did you tested if this is needed?

Reply:
Same in sparklineChartDescriptor.ts / staleDataDescriptor.ts / subtitleBaseDescriptor.ts / textFormattingDescriptor.ts